Feat: x-pack cloud id/auth for monitoring/management (#11496)
elastic/logstash: docs/static/settings/monitoring-settings.asciidoc

`xpack.monitoring.elasticsearch.ssl.keystore.password`::

Optional settings that provide the password to the keystore.

[[monitoring-additional-settings]]
===== Additional settings

`xpack.monitoring.elasticsearch.cloud_id`::

> If you're using Elasticsearch cloud, you should specify the identifier here.

Reviewer: Similar to other comment: `s/Elasticsearch cloud/{es} in Elastic Cloud/g`

```suggestion
If you're using {es} in Elastic Cloud, you should specify the identifier here.
```

Reviewer 2: Should be {es} in {ecloud}

This is an alternative to specifying `xpack.monitoring.elasticsearch.hosts`,

> if `cloud_id` is set having also a `hosts` configuration makes no sense.

Reviewer 3: Same comment as above

> if `cloud_id` is set having also a `hosts` configuration makes no sense.

Reviewer: Similar to other comment, needs rewording.

```suggestion
which should not be used if `cloud_id` is configured.
```

The {es} instances that you want to ship your Logstash metrics to. This might be
the same {es} instance specified in the `outputs` section in your Logstash
configuration, or a different one.

`xpack.monitoring.elasticsearch.cloud_auth`::

> If you're using Elasticsearch cloud, you can set your auth credentials here.

Reviewer 2: ```suggestion
If you're using {es} on {ecloud}, you can set your auth credentials here.
```
